package.json

```diff
@@ -19,21 +19,27 @@
     "prepush": "yarn lint && yarn build:dist && yarn jest --changedSince main"
   },
   "devDependencies": {
-    "@types/jest": "^27.5.1",
+    "@babel/core": "^7.22.9",
+    "@babel/preset-env": "^7.22.9",
+    "@babel/preset-typescript": "^7.22.5",
+    "@types/jest": "^29.5.3",
     "@types/node": "^18",
     "@typescript-eslint/eslint-plugin": "^4.29.3",
     "@typescript-eslint/parser": "^4.29.3",
     "eslint": "^7.24.0",
     "eslint-config-prettier": "^8.2.0",
     "eslint-plugin-jest": "^24.3.5",
     "husky": "^4.2.5",
-    "jest": "^27.5.1",
-    "jest-extended": "^0.11.5",
+    "jest": "^29.6.1",
+    "jest-extended": "^4.0.0",
     "lerna": "^6.0.0",
     "lint-staged": "^10.2.6",
     "prettier": "^2.2.1",
-    "ts-jest": "^27.1.5",
+    "ts-jest": "^29.1.1",
     "typescript": "^4.3.5"
   },
+  "resolutions": {
+    "@types/node": "^18"
```

Comment:
I found that one of the packages was installing a different version of Node (admittedly, I don't remember which one). Needed to explicitly add a resolution. As we upgrade more packages, this should be able to be removed.